Lib/test/test_exceptions.py

```diff
@@ -1187,7 +1190,8 @@ def inner():
         else:
             self.fail("RecursionError not raised")
         self.assertEqual(wr(), None)
-
+    
+    @unittest.skipIf(sys.platform == 'win32', 'error specific to cpython')
```

Comment:
What do you mean by "error specific to cpython"?

Reply:
> Issue #12802: "not a directory" errors are ENOTDIR even on Windows 

It seems like CPython has a bug that errno returns `ENOTDIR` on Windows.